### PR TITLE
Docs: how to get latest snapshot of incremental append data & vacuuming

### PR DESCRIPTION
**Pull request by sherifnada**
Add a section to the docs about the recommended approach for obtaining the latest snapshot of data in an incremental append sync and mention that vacuuming is necessary.

### CODE REVIEW

File: docs/architecture/incremental.md [hunk 1]
```diff
@@ -84,7 +84,17 @@ Some sources cannot define the cursor without user input. For example, in the [p
 
 \(You can find a more technical details about the configuration data model [here](catalog.md)\).
 
+## Getting the Latest Snapshot of data
+
+As demonstrated in the examples above, with **Incremental Append,** a record which was updated in the source will be appended to the destination rather than updated in-place. This means that if data in the source uses a primary key \(e.g: `user_id` in the `users` table\), then the destination will end up having multiple records with the same primary key value. 
+
+However, some use cases require only the latest snapshot of the data. If you want the latest snapshot and are syncing to a destination that supports views, we recommend creating a view on your data which groups by the primary key and deduplicates by the largest `_airbyte_emitted_at` values. The `_airbyte_emitted_at`  column is added by Airbyte to all records synced to the destination.
```
Comment (cgardens) [inline docs/architecture/incremental.md:91]: would it be valuable to write the query for an imaginary case? this is an annoying query to write. i think as a reader i would appreciate an example that i could copy and paste and the modify.

File: docs/architecture/incremental.md [hunk 2]
```diff
@@ -84,7 +84,17 @@ Some sources cannot define the cursor without user input. For example, in the [p
...
+
+As demonstrated in the examples above, with **Incremental Append,** a record which was updated in the source will be appended to the destination rather than updated in-place. This means that if data in the source uses a primary key \(e.g: `user_id` in the `users` table\), then the destination will end up having multiple records with the same primary key value. 
+
+However, some use cases require only the latest snapshot of the data. If you want the latest snapshot and are syncing to a destination that supports views, we recommend creating a view on your data which groups by the primary key and deduplicates by the largest `_airbyte_emitted_at` values. The `_airbyte_emitted_at`  column is added by Airbyte to all records synced to the destination. 
+
+{% hint style="info" %}
+Note that in **Incremental Append,** the size of the data in your warehouse increases monotonically since an updated record in the source is appended to the destination rather than updated in-place. If you only care about having the latest snapshot of your data, you may want to periodically run "vacuum" jobs which retain only the latest instance of each record, deduping by primary key.
```
Comment (cgardens) [inline docs/architecture/incremental.md:94]: i don't think this is technically a correct use of the concept of database vacuum. maybe just call them clean up jobs?not a big deal if you want to keep it, but i read that and it stood out as not quite accurate.

**Review by cgardens — APPROVED** [review]

File: docs/architecture/incremental.md (hunk 1 above)
Comment (ChristopheDuong) [reply docs/architecture/incremental.md:91]: Like it is done here?

https://docs.airbyte.io/tutorials/connecting-el-with-t-using-sql#customize-sql-query

Comment (sherifnada) [reply docs/architecture/incremental.md]: added a simple(r?) query here and also pointed to the docs

**Pull request merged**